backend/controllers/movieController.js

```diff
@@ -27,8 +29,44 @@ const getMovieEntry = async (req, res, next) => {
   } catch (err) { next(err); }
 };
 
+const addComment = async (req, res, next) => {
+  try {
+    const newComment = {
+      userId: req.body.userId,
```

Comment:
I looked at our documentation, and realized, that actually we don't need to send in req.body userId, because only logged-in users can leave comments. So, we should update our documentation and  get user id by 
_userId = req.user;_